ada_2_fhir-r4/zibs2020/payload/zibs2020_2_fhir.xsl

```diff
@@ -57,10 +59,11 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     <xd:doc>
         <xd:desc>Start conversion.</xd:desc>
     </xd:doc>
-    <xsl:template match="/">
+    <!-- TODO: Had to silence this to make the driver work -->
+<!--    <xsl:template match="/">
```

Comment:
Sorry, kladje. Zoals gezegd weet ik even niet of dit de juiste plek is om een override voor een specifieke template te plaatsen.